lib/rbs/definition_builder.rb

```diff
@@ -239,6 +239,19 @@ def build_singleton0(type_name)
             definition.class_variables.merge!(defn.class_variables)
           end
 
+          one_ancestors.each_included_module do |mod|
```

Comment:
How about using `#instance_ancestors` explicitly, instead of having `included_modules` for _singleton_ ancestors?